api/tests/identity-access-management/unit/domain/usecases/create-oidc-user.usecase.test.js

```diff
@@ -137,4 +137,49 @@ describe('Unit | Identity Access Management | Domain | UseCase | create-oidc-use
       logoutUrlUUID: 'logoutUrlUUID',
     });
   });
+
+  it('creates the user account with given language', async function () {
+    // given
+    const idToken = 'idToken';
+    const language = 'nl';
+    authenticationSessionService.getByKey.withArgs('AUTHENTICATION_KEY').resolves({
+      sessionContent: { idToken, accessToken: 'accessToken' },
+      userInfo: { firstName: 'Jean', lastName: 'Heymar', externalIdentityId: 'externalId' },
+    });
+    authenticationMethodRepository.findOneByExternalIdentifierAndIdentityProvider
+      .withArgs({ externalIdentifier: 'externalId', identityProvider: 'SOME_IDP' })
+      .resolves(null);
+    oidcAuthenticationService.createUserAccount.resolves(10);
+    oidcAuthenticationService.createAccessToken.withArgs(10).returns('accessTokenForExistingExternalUser');
+    oidcAuthenticationService.saveIdToken.withArgs({ idToken, userId: 10 }).resolves('logoutUrlUUID');
+
+    // when
+    await createOidcUser({
+      identityProvider: 'SOME_IDP',
+      authenticationKey: 'AUTHENTICATION_KEY',
+      localeFromCookie: 'nl-BE',
+      language,
+      authenticationSessionService,
+      oidcAuthenticationServiceRegistry,
+      authenticationMethodRepository,
+      userToCreateRepository,
+      userLoginRepository,
+    });
+
+    // then
+    expect(oidcAuthenticationService.createUserAccount).to.have.been.calledWithMatch({
+      user: {
+        firstName: 'Jean',
+        lastName: 'Heymar',
+        lang: 'nl',
+        locale: 'nl-BE',
+        cgu: true,
+        lastTermsOfServiceValidatedAt: now,
+      },
+      sessionContent: { idToken, accessToken: 'accessToken' },
+      externalIdentityId: 'externalId',
+      userToCreateRepository,
+      authenticationMethodRepository,
+    });
+  });
```

Comment:
Plutôt que créer un nouveau test, adapter le test `creates the user account and returns an access token, the logout url uuid and update the last logged date with the existing external user id` parce qu'il doit être mis à jour car `createOidcUser` sera toujours appelé avec le nouveau paramètre `language`.